client/navigation/utils.ts

```diff
@@ -177,45 +210,65 @@ export const sortMenuItems = ( menuItems ) => {
  * @return {Object} Mapped menu items and categories.
  */
 export const getMappedItemsCategories = (
-	menuItems,
-	currentUserCan = null
+	menuItems: Array< Item >,
+	currentUserCan: Function | null = null
```

Comment:
Another situation where a type sig for the function would probably be quite helpful.

if it makes the type declaration too long then you can just do something like:

`type CurrentUserCallback = (someArg: SomeType) => void`

`currentUserCan?: CurrentUserCallback`